translations/de-DE.json

```diff
@@ -0,0 +1,267 @@
+    "dashboard.howToStartWorkingOnNewDashboardEditLinkAriaLabel": "Dashboard bearbeiten",
+    "dashboard.howToStartWorkingOnNewDashboardEditLinkText": "Bearbeiten",
+    "dashboard.listing. savedObjectWarning": "Nach Titel kann nicht gefiltert werden",
+    "dashboard.listing.createButtonText": "Erstellen Sie",
+    "dashboard.listing.createNewDashboard.combineDataViewFromOpenSearchDashboardsAppDescription": "Sie können Datenansichten aus jeder OpenSearch-Dashboards-App in einem Dashboard kombinieren und alles an einem Ort sehen.",
+    "dashboard.listing.createNewDashboard.createButtonLabel": "Neues Dashboard erstellen",
+    "dashboard.listing.createNewDashboard.newToOpenSearchDashboardsDescription": "Neu bei OpenSearch Dashboards? Öffnen Sie {SampleDataInstallLink} für einen Test.",
+    "dashboard.listing.createNewDashboard.sampleDataInstallLinkText": "Installieren Sie einige Beispieldaten",
+    "dashboard.listing.createNewDashboard.title": "Erstellen Sie Ihr erstes Dashboard",
+    "dashboard.listing.dashboardsTitle": "Dashboards",
+    "dashboard.listing.noItemsMessage": "Sieht aus, als hätten Sie keine Dashboards.",
+    "dashboard.listing.table.columnUpdatedAtDescription": "Letzte Aktualisierung des gespeicherten Objekts",
+    "dashboard.listing.table.columnUpdatedAtName": "Letzte Aktualisierung",
+    "dashboard.listing.table.descriptionColumnName": "Beschreibung",
+    "dashboard.listing.table.entityName": "Dashboard",
+    "dashboard.listing.table.entityNamePlural": "Dashboards,",
+    "dashboard.listing.table.titleColumnName": "Position",
+    "dashboard.listing.table.typeColumnName": "Typ",
+    "dashboard.panel.AddToLibrary": "Zur Bibliothek hinzufügen",
+    "dashboard.panel.clonedToast": "Geklonter Bereich",
+    "dashboard.panel.clonePanel": "Bereich Klonen",
+    "dashboard.panel.invalidData": "Ungültige Daten in der URL",
+    "dashboard.panel.LibraryNotification": "Bibliothek",
+    "dashboard.panel.libraryNotification.toolTip": "Dieses Fenster ist mit einem Bibliothekselement verknüpft. Die Bearbeitung des Bereichs kann sich auf andere Dashboards auswirken.",
+    "dashboard.panel.removePanel.replacePanel": "Bereich austauschen",
+    "dashboard.panel.title.clonedTag": "Text",
+    "dashboard.panel.unableToMigratePanelDataForSixOneZeroErrorMessage": "Bereichsdaten können nicht migriert werden, um die Abwärtskompatibilität von „6.1.0“ zu gewährleisten. Der Bereich enthält keine erwarteten Spalten- und/oder Zeilenfelder",
+    "dashboard.panel.unableToMigratePanelDataForSixThreeZeroErrorMessage": "Die Bereichsdaten können nicht für die Abwärtskompatibilität von „6.3.0\" migriert werden. Der Bereich enthält nicht das erwartete Feld: {key}",
+    "dashboard.panel.unlinkFromLibrary": "Verknüpfung mit Bibliothekselement aufheben",
+    "dashboard.placeholder.factory.displayName": "Platzhalter",
```

Comment:
"Anzeigename"

Reply:
It is plain `placeholder`:
```
    return i18n.translate('dashboard.placeholder.factory.displayName', {
      defaultMessage: 'placeholder',
    });
```
So the capitalization is wrong too.

Reply:
All nouns are capitalized in German, it's a spelling error to write Platzhalter in lowercase